bin/transform

```diff
@@ -294,7 +395,15 @@ if __name__ == '__main__':
     gisaid_data = parse_authors(gisaid_data)
     gisaid_data = parse_age(gisaid_data)
     gisaid_data = parse_sex(gisaid_data)
-    gisaid_data = update_metadata(gisaid_data)
+
+    # Write intermediate metadata to file for comparison before annotations
+    gisaid_data.to_csv(base / 'data/draft_metadata.tsv', sep='\t', index=False)
```

Comment:
This file path seems like it should be based on the `args.output_metadata` path rather than hardcoded.

Reply:
Yes, that makes sense.